src/Umbraco.Core/Constants-SystemDirectories.cs

```diff
@@ -60,6 +60,7 @@ public static class SystemDirectories
         /// <summary>
         ///     The default folder where Umbraco log files are stored
         /// </summary>
+        [Obsolete("Use LoggingSettings.GetLoggingDirectory() instead")]
```

Comment:
We should mention what version we want to remove this (also as a note for ourselves)
```suggestion
        [Obsolete("Use LoggingSettings.GetLoggingDirectory() instead, will be removed in Umbraco 13.")]
```